backoffice/src/Views/Letter/View.cshtml

```diff
@@ -62,6 +62,14 @@
         
                 </td>
             </tr>
+            <tr>
+                <th scope="row">Is QC Approved</th>
+                <td>@Html.DisplayFor(m => m.IsQualityAssured)</td>
+            </tr>
+            <tr>
+                <th scope="row">QC Approved By</th>
+                <td>@Html.DisplayFor(m => m.QualityAssuredUser)</td>
```

Comment:
In the model, might be worth defaulting this to something if there isn't one, or just having it empty? unsure.